kedro/contrib/io/pyspark/spark_jdbc.py

```diff
@@ -120,7 +120,7 @@ def __init__(
                 https://spark.apache.org/docs/latest/api/python/pyspark.sql.html?highlight=jdbc#pyspark.sql.DataFrameWriter.jdbc
 
         Raises:
-            DataSetError: When either ``url`` or ``table`` is empty.
+            DataSetError: When either ``url`` or ``table`` is empty or when a property is provided with a None value.
```

Comment:
You'll need to wrap this onto another line, as it exceeds the max line length the linter allows (100 characters). Better yet, I think it would make sense to wrap it at 72 characters, as recommended by PEP-8.

Reply:
Thanks for the note. My PyCharm still had its PEP8 line length still set incorrectly; should have added proper PEP8 guide lines when originally changing code.